### ISSUE TITLE
Prettier wraps long line

### ISSUE THREAD

**Issue by hoest** [shannonmoeller/stylelint-config-prettier#13]
I'm using `stylinelint` in SublimeText with `stylelint-config-standard` and `stylelint-config-prettier`. I have this issue:

![image](https://user-images.githubusercontent.com/606546/37585570-dae36554-2b59-11e8-873c-3c277aa0839a.png)

Prettier wraps the long line, but Stylelint does not like this... What can I do to fix this?

**Comment by shannonmoeller** [comment]
May I see your stylelint config? We're specifically disabling the `indentation` rule, so something seems fishy.

**Comment by hoest** [comment]
This is my `.stylelintrc.yml`:

```yml
---
extends:
- stylelint-config-standard
- stylelint-config-prettier

rules:
  indentation: 2
  function-url-quotes: always
  property-no-vendor-prefix: true
  value-no-vendor-prefix: true
```

**Comment by shannonmoeller** [comment]
Thanks! You don't need the `indentation: 2` rule as Prettier will take care of that for you. If you need something other than prettier's default indentation, you should use the `tabWidth` option in your [prettier config](https://prettier.io/docs/en/options.html).

**Comment by shannonmoeller** [comment]
You can use [`stylelint-config-prettier-check`](https://github.com/shannonmoeller/stylelint-config-prettier#cli-helper-tool) to see if your configuration contains incompatible values.

**Comment by shannonmoeller** [comment]
Did this resolve your issue?

**Comment by hoest** [comment]
Hi,

No, I have removed `indentation: 2`, but still the same message:

![image](https://user-images.githubusercontent.com/606546/37641610-bd0880d4-2c1a-11e8-82ae-7a055948696b.png)

```
$ yarn stylelint-check
yarn run v1.5.1
$ stylelint-config-prettier-check
No conflicting rules detected in your stylelint configuration!
Done in 1.85s.

$ yarn stylelint
yarn run v1.5.1
$ stylelint --config .stylelintrc.yml --fix src/**/*.css

src/assets/styling.css
 585:24  ×  Unexpected "   "   selector-descendant-combinator-no-non-space
 586:32  ×  Unexpected "   "   selector-descendant-combinator-no-non-space

$ cat .\.stylelintrc.yml
---
extends:
- stylelint-config-standard
- stylelint-config-prettier

rules:
  function-url-quotes: always
  property-no-vendor-prefix: true
  value-no-vendor-prefix: true
```

**Comment by shannonmoeller** [comment]
Looks like we should add `selector-descendant-combinator-no-non-space` to the list of rules that are turned off.

**Comment by shannonmoeller** [comment]
`selector-descendant-combinator-no-non-space` set to `null` and released as v3.1.0
